spartan/scripts/prepare_sepolia_accounts.sh

```diff
@@ -56,35 +79,67 @@ if ! command -v yq &>/dev/null; then
   chmod +x /usr/local/bin/yq
 fi
 
-# Create a new mnemonic with the required number of accounts
+# Create a new mnemonic
 echo "Creating mnemonic..."
-cast wallet new-mnemonic --accounts "$num_accounts" --json >"$tmp_filename"
+cast wallet new-mnemonic --json >"$tmp_filename"
 MNEMONIC=$(jq -r '.mnemonic' "$tmp_filename")
-ADDRESSES=$(jq -r '.accounts[].address' "$tmp_filename")
 
-# Convert ETH to wei
-wei_amount=$(cast to-wei "$eth_amount" ether)
+echo "MNEMONIC: $MNEMONIC"
 
-# Get current gas price and add 25% buffer
-echo "Getting gas price..."
-gas_price=$(cast gas-price --rpc-url "$ETHEREUM_HOST")
-gas_price=$((gas_price * 125 / 100)) # Add 25% to gas price
+# Cast has a limit of 255 accounts per mnemonic command
+# We'll need to derive accounts in batches
+echo "Deriving $total_accounts accounts from mnemonic..."
```

Comment:
To double check: `total_accounts` are never actually used, right? Especially considering we have max indices of 3000.

Reply:
ah yes, I think I added `total_accounts` when trying to do all accounts at once & then realized you can't go further than 255 with the cast command. Will remove, it's not even the right number of actual accounts we fund